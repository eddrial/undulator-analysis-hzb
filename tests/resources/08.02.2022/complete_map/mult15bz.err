 error report for the data set mult15bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1590 data points 
        out of totally   2558 data points
 the derivative at  0.56583E+03 and  0.58991E+03 is zero 
 the derivative at  0.58991E+03 and  0.62076E+03 is zero 
 the derivative at  0.64696E+03 and  0.70273E+03 is zero 
 the derivative at  0.10382E+04 and  0.10602E+04 is zero 
 the derivative at  0.10602E+04 and  0.10927E+04 is zero 
 the derivative at  0.11185E+04 and  0.11506E+04 is zero 
 the derivative at  0.11785E+04 and  0.12094E+04 is zero 
 the derivative at  0.12094E+04 and  0.12580E+04 is zero 
 the derivative at  0.12580E+04 and  0.12896E+04 is zero 
 the derivative at  0.14865E+04 and  0.15191E+04 is zero 
 the derivative at  0.15466E+04 and  0.15715E+04 is zero 
 the derivative at  0.17397E+04 and  0.17709E+04 is zero 
 the derivative at  0.18237E+04 and  0.18478E+04 is zero 
 the derivative at  0.18478E+04 and  0.18829E+04 is zero 
 the derivative at  0.18829E+04 and  0.19074E+04 is zero 
 the derivative at  0.21052E+04 and  0.21288E+04 is zero 
 the rms value of the residuals is  0.22774E-02
 the maximum residual is  0.68205E-02
 the rms value of the lambda/2-filtered data is  0.12607E-02
 the maximum value of the lambda/2-filtered data is  0.48919E-02
