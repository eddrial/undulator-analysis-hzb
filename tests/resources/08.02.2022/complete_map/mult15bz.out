mult15bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -8.0853608E-03
   2 -1.6736003E-04
   3  1.9722366E-04
   4 -5.1526540E-06
   5 -4.4217818E-06
   6  5.6020554E+01
   7  1.0392294E+03
   8  1.0488636E+03
   9  1.0431730E+03
  10  1.5638028E+00
  11 -1.5846833E+00
 number of iterations:   13
     sumsq:  8.24668E-03
       rms:  2.27741E-03  dimension of magnetic field
       rms:  2.81661E+01  percent of b_eff
   rms_ext:  2.42973E+01  percent of b_eff
 B_eff =   8.0856E-03
 B_max =  -8.0350E-03
 K_eff =    0.042
 lamda_1 (nm) =      2.54255

        eigenvectors in parameter space

 error                         direction
    20.03,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
    18.16,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     1.44,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.73,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.10,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00,  0.66 -0.46 -0.40 -0.13  0.42  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.26  0.69 -0.63  0.09 -0.22  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.44  0.07 -0.26  0.53  0.67  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.34 -0.20  0.17  0.83 -0.34  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.43  0.52  0.58  0.01  0.45  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:   0.06657 Tmm
 kick_2:  -0.06657 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.45132E+01
      
 the truncated data set has only   1590 data points 
      
 the rms value of the residuals is  0.22774E-02
 the maximum residual is  0.68205E-02
      
 the rms value of the lambda/2-filtered data is  0.12607E-02
 the maximum value of the lambda/2-filtered data is  0.48919E-02
      
 the rms phase error (all data) is    1.631 deg.
 the maximum phase error (all data) is    3.459 deg.
      
 the rms phase error (extrema) is    5.588 deg.
 the maximum phase error (extrema) is   10.302 deg.
      
 the Walker-rms phase error (all data) is    1.631 deg.
 the Walker-maximum phase error (all data) is    3.450 deg.
      
 the Walker-rms phase error (extrema) is    1.598 deg.
 the Walker-maximum phase error (extrema) is    3.482 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             1.630703    
 local phase error            0.4967569    
 slowly varying phase error  NaN           
 oscillatory phase error     NaN           
 -----------------------------------------
